tests/system/providers/apache/pinot/example_pinot_dag.py

```diff
@@ -0,0 +1,57 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""Example DAG demonstrating the usage of the PinotAdminHook and PinotDbApiHook."""
+
+from __future__ import annotations
+
+from datetime import datetime
+
+from airflow import DAG
+from airflow.decorators import task
+from airflow.providers.apache.pinot.hooks.pinot import PinotAdminHook, PinotDbApiHook
+
+with DAG(
+    dag_id="example_pinot_hook",
+    schedule_interval=None,
```

Comment:
```suggestion
    schedule=None,
```
As of 2.4., the `schedule_interval` has been deprecated for the more general `schedule` parameter. [Here are the release notes](https://airflow.apache.org/docs/apache-airflow/2.4.0/release_notes.html#deprecation-of-schedule-interval-and-timetable-arguments-25410) for reference.